bees/telegrambee/telegrambee.go

```diff
@@ -116,6 +117,11 @@ func (mod *TelegramBee) Run(eventChan chan bees.Event) {
 						Type:  "string",
 						Value: strconv.Itoa(update.Message.From.ID),
 					},
+					{
+						Name:  "timestamp",
+						Type:  "time.Time",
```

Comment:
I'd keep this simple and still call the type `timestamp`. Its internal Go type is separate to that type definition here, which we mostly use for the admin interface.

Reply:
Done.